qtm/qrt.py

```diff
@@ -303,14 +299,31 @@ def has_transport(self):
             timeout=self._timeout,
         )
 
-        # TODO GetCaptureC3D
-        # TODO GetCaptureQTM
+    async def calibrate(self, timeout=600):  # Timeout 10 min.
```

Comment:
I guess some kind of test should be created for this..?

Reply:
I have added tests for calibration command.